src/utilities.h

```diff
@@ -127,6 +127,7 @@
 # define HAVE_STACKTRACE
 #endif
 
+#ifndef HAVE_SYMBOLIZE
 // defined by gcc
 #if defined(__ELF__) && defined(OS_LINUX)
 # define HAVE_SYMBOLIZE
```

Comment:
I kind of wish we could move this check into configuration time, but then we'd need to do it in a couple build systems. Oh well.